belgif-rest-problem/src/main/java/io/github/belgif/rest/problem/api/InputValidationIssues.java

```diff
@@ -108,10 +108,10 @@ public static InputValidationIssue rejectedInput(InEnum in, String name, Object
                 .in(in, name, value);
     }
 
```

Comment:
The class also contains `ISSUE_TYPE_UNKNOWN_INPUT` but it isn't used in a method nor does it have a test. Is that normal?

Reply:
It's a standardized issue type at Belgif (https://www.belgif.be/specification/rest/api-guide/#rule-req-valid) which we didn't use at CBSS so far.

I added the corresponding InputValidationIssues.unknownInput() method and a unit test.